.replit

```diff
@@ -0,0 +1,2 @@
+language = "nodejs"
+run = "cd examples && node art.js"
```

Comment:
Shouldn't it be: `npm install && cd examples && node art.js`

At least, as I tested it on repl.it, it came just cloned, but with no dependencies installed

Reply:
oh yes, nice catch - i'll make that change